staging/src/k8s.io/kube-scheduler/config/v1beta1/types.go

```diff
@@ -74,6 +74,9 @@ type KubeSchedulerConfiguration struct {
 	// nodes will be scored.
 	PercentageOfNodesToScore *int32 `json:"percentageOfNodesToScore,omitempty"`
 
+	// UnschedulableQTimeInterval is the max duration for pods in unschedulableQ
+	UnschedulableQTimeInterval *int64 `json:"unschedulableQTimeInterval,omitempty"`
```

Comment:
The name should follow the naming convention: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#naming-conventions

Reply:
Hi Wei, thanks so much for your comments here :) 

From the above naming convention doc, how about let's change this param to `UnschedulableQTimeoutSeconds`? 

Jerry

Reply:
lets name it similar to `PodMaxBackoffSeconds`: `PodMaxUnschedulableSeconds `, also, please update the comment to mention the default value, something like:

```
PodMaxUnschedulableSeconds is the max time a pod spends in unschedulable status.
The default value is "60s".
```

Reply:
Good idea, will update the name like that!